base-template.sh

```diff
@@ -471,7 +473,7 @@ update_shared_hooks_if_appropriate() {
 
             if [ -d "$SHARED_ROOT/.git" ]; then
                 echo "* Updating shared hooks from: $SHARED_REPO"
-                PULL_OUTPUT=$(cd "$SHARED_ROOT" && git pull 2>&1)
+                PULL_OUTPUT=$(git --work-tree="$SHARED_ROOT" --git-dir="$SHARED_ROOT/.git" pull 2>&1)
```

Comment:
Perhaps `PULL_OUTPUT=$(cd "$SHARED_ROOT" && git -c core.hooksPath=/dev/null pull 2>&1)` would work too?

Reply:
Sure, but my changes are not due to the hooks beeing executed-> its because it needs `unset GIT_DIR` or the solution currently done.

`-c core.hooksPath=/dev/null` is certainly good